Kolorofon/Scenes/StartMenu/Tutorial/TutorialViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TutorialViewController.swift
+//  Kolorofon
+//
+//  Created by Marek on 03/04/2021.
+//  Copyright © 2021 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+class TutorialViewController: UIViewController {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/Kolorofon/blob/3742950033d538b87c00f082cb935a9c54176b8f/Kolorofon/Scenes/StartMenu/Tutorial/TutorialViewController.swift#L11"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>